<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeachProcess</class>
 <widget class="QDialog" name="TeachProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Обучение</string>
  </property>
  <layout class="QGridLayout" name="_grid_layout">
   <item row="0" column="0">
    <widget class="QLabel" name="_label_teach">
     <property name="text">
      <string>Обучить выборке</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QDialogButtonBox" name="_button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="_box_count">
     <property name="suffix">
      <string> раз</string>
     </property>
     <property name="prefix">
      <string/>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="_label_speed">
     <property name="text">
      <string>Скорость обучения</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="_speed_box">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_button_box</sender>
   <signal>accepted()</signal>
   <receiver>TeachProcess</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_button_box</sender>
   <signal>rejected()</signal>
   <receiver>TeachProcess</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
